README.md

```diff
@@ -95,10 +95,10 @@ https://forms.gle/u1RXnbocbFWqfxGb9
 
 ```bash
 # run the following command in 1 terminal
-docker-compose up -f docker-compose.dev.yml --force-recreate --build
+docker-compose --file ./docker-compose.dev.yml up --force-recreate --build
 
 # and the following in a different terminal
-cd web && ng build --output-path ./dist --base-href "/" --prod --watch
+cd web && npm install && ng serve
```

Comment:
Using `ng serve` here instead of `ng build` will give you the local development server with automatic code watch and browser reload on changes. `npm install` is required the first time to install dependencies.